spyder/plugins/variableexplorer/widgets/collectionsdelegate.py

```diff
@@ -135,16 +135,17 @@ def createEditor(self, parent, option, index, object_explorer=False):
                     message.format(val_type=val_type, module=module))
                 return
             else:
-                message = _("Spyder is unable to show the variable you're"
-                            " trying to view because the module "
-                            "<tt>{module}</tt> is not ")
                 if running_in_mac_app():
```

Comment:
```suggestion
                if running_in_mac_app() or is_pynsist():
```

`is_pynsist()` detects if Spyder is running in the Windows app. And you can import from `spyder.config.base`.